operations/patch_util.go

```diff
@@ -584,7 +584,7 @@ func getGenericPatchDisplay(ac *legacyClient, params outputPatchParams) (string,
 		}{
 			Patch:             &p,
 			ShowSummary:       params.summarize,
-			ShowFinalized:     p.IsCommitQueuePatch(),
```

Comment:
Does this part of the patch display template still apply for merge queue patches? From the patch display template, I would have thought showing finalized/not finalized doesn't really matter for merge queue patches since users don't finalize the patch.